modules/react/tabs/lib/TabsPanels.tsx

```diff
@@ -5,11 +5,11 @@ import {useListRenderItems} from '@workday/canvas-kit-react/collection';
 
 import {useTabsModel} from './useTabsModel';
 
-export interface TabsPanelsProps<T = unknown> {
+export interface TabsPanelsProps {
   /**
    *
    */
-  children: ((item: T) => React.ReactNode) | React.ReactNode;
+  children: ((item: any) => React.ReactNode) | React.ReactNode;
```

Comment:
I think Nicholas was trying to avoid having type `any` here because `T` would give you a generic of the item, but not it just says any. I would sync up with him on this one

Reply:
This one should be reviewed, because it cause type mismatch for all menu components. It's possible to set generic type for `List` items, but not for `Menu.Popper` part of components.